pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -107466,7 +107466,7 @@ self: {
        platforms = [
          "aarch64-linux" "armv7l-linux" "i686-linux" "x86_64-linux"
        ];
-     }) {inherit (pkgs.gnome3) gnome-keyring; 
+     }) {inherit (pkgs.gnome) gnome-keyring; 
```

Comment:
@sternenseemann Do we need to change something so that we don‘t override this?

Reply:
IIRC, it should just find the correct package somehow. https://github.com/NixOS/cabal2nix/blob/master/src/Distribution/Nixpkgs/Haskell/FromCabal.hs